site/typedoc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "entryPoints": ["../src/modules/*" ],
+  "entryPointStrategy": "expand",
+  "cleanOutputDir": true,
+  "excludeExternals": true,
+  "name": "IaSQL documentation",
+  "categorizeByGroup": false,
+  "disableSources": true,
+  "readme": "./sql_reference.md",
```

Comment:
do we have this file?

Reply:
this is actually not used. The plugin just checks if the readme is set to some value or to "none"